.travis.yml

```diff
@@ -1,11 +1,20 @@
+dist: trusty
 sudo: false
-language: ruby
-rvm: 2.1.9
-addons:
-  apt:
-    packages:
-    - yajl-tools
-install:
-  - pip install --user jenkins-job-builder
-  - (cd puppet && bundle install)
-script: ./.travis.sh
+matrix:
+  include:
+    - language: ruby
+      rvm:
+        - 2.1.9
+      install: (cd puppet && bundle install)
+      script: ./.travis/puppet.sh
+      addons:
+        apt:
+          packages:
+            - yajl-tools
+    - language: python
+      python:
+        - "2.7"
```

Comment:
Any reason this is in quotes and `2.1.9` above isn't?

Reply:
Otherwise it's intepreted as a floating point rather than a string. IIRC weird things happen and the travis docs recommend it this way. It's the same reason we write ruby 2.0.0 in other JJB jobs rather than 2.0